src/SFA.DAS.AssessorService.Web/Views/ManageUsers/Index.cshtml

```diff
@@ -31,29 +31,30 @@
                         </tr>
                     </thead>
                     <tbody>
-                        @foreach (var item in Model)
+                        @foreach (var contact in Model)
                         {
                             <tr>
-                                <td data-label="@Localizer["User details"]" class="nowrap" >
-                                    @Html.DisplayFor(modelItem => item.Contact.DisplayName)<br />
-                                    <span>@Html.DisplayFor(modelItem => item.Contact.Email)</span>
+                                <td data-label="@Localizer["User details"]" class="nowrap">
+                                    <a asp-controller="ManageUsers" asp-action="User" asp-route-userid="@contact.Contact.Id">@Html.DisplayFor(modelItem => contact.Contact.DisplayName)</a>
```

Comment:
This will show the new link correctly when the css upgrade is merged:
```suggestion
                                    <a class="govuk-link" asp-controller="ManageUsers" asp-action="User" asp-route-userid="@contact.Contact.Id">@Html.DisplayFor(modelItem => contact.Contact.DisplayName)</a>
```